packages/hdwallet-native/src/crypto/EncryptedWallet.test.ts

```diff
@@ -120,10 +120,42 @@ describe("EncryptedWallet", () => {
       await expect(result.decrypt()).resolves.toEqual(PLAIN);
     });
 
-    it("should require the wallet to be initialized", async () => {
+    it("should require that the wallet to be initialized", async () => {
```

Comment:
`that the wallet to be initialized` doesn't sound right

Reply:
ah damn.